src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1342,6 +1340,12 @@ static ImmutableHashSet<ISymbol> computeGeneratedCodeSymbolsInTree(SyntaxTree tr
                     declarationInfoBuilder.Free();
                     return generatedSymbolsBuilder != null ? generatedSymbolsBuilder.ToImmutable() : ImmutableHashSet<ISymbol>.Empty;
                 }
+
+                static bool containsGeneratedCodeToken(SyntaxNode root)
+                {
+                    return root.DescendantTokens().Any(static token => string.Equals(token.ValueText, "GeneratedCode", StringComparison.Ordinal) ||
+                                                                       string.Equals(token.ValueText, nameof(GeneratedCodeAttribute), StringComparison.Ordinal));
```

Comment:
>nameof(GeneratedCodeAttribute) [](http://example.com/codeflow?start=102&length=30)

Consider using `"GeneratedCodeAttribute"` explicitly instead since `"GeneratedCode"` is already used above, and since it will allow removing the additional `using` statement.

Reply:
I think I'm going to pass on this. If you feel strongly, I can do in a followup PR. Strongly referencing this class was done previously (in the deleted AnalyzerDriver.GeneratedCodeTokenWalker.cs file).

Reply:
Actually, never mind, since this check was just moved from `GeneratedCodeTokenWalker`.